src/MoBi.Presentation/Presenter/EditSimulationPresenter.cs

```diff
@@ -40,7 +39,7 @@ public class EditSimulationPresenter : SingleStartPresenter<IEditSimulationView,
       private bool _diagramLoaded;
       private readonly IHeavyWorkManager _heavyWorkManager;
       private readonly IChartFactory _chartFactory;
-      private bool _dataShown;
+      private bool _chartLoaded;
```

Comment:
why do we still need this flag? i thought we said this was not required anymore and we should load the chart on Edit?